packages/client/CHANGELOG.md

```diff
@@ -2,11 +2,15 @@
 
 All notable changes to this npm module are documented in this file.
 
+## [4.0.1] - 2022-09-14
+
+- monaco-converter removal [#419](https://github.com/TypeFox/monaco-languageclient/issues/419): `monaco-converter` has been re-introduced to help dependent projects adapt
```

Comment:
`removal [...] has been re-introduced` seems a little awkward to me, I'm not sure if it was removed and put back